src/index.ts

```diff
@@ -0,0 +1,17 @@
+import { getCookie } from '@ketch-com/ketch-cookie'
+
+/**
+ * Get current consent state from _swb_ketch_ cookie.
+ * @param w The window object
+ */
+export default function getConsent(w: Window): any {
+  const value = getCookie(w, '_swb_ketch_')
+  if (value) {
+    try {
+      return JSON.parse(atob(value))
```

Comment:
I think we may just want to return the purpose to true false key value pairs so the interface is not directly tied to the contents of the cookie.

Thoughts @sethyates ?